apps/prairielearn/src/pages/partials/groupWorkInfoContainer.ejs

```diff
@@ -9,7 +9,7 @@
       <p>
         <% if (groupConfig.minimum > 1) { %>
         This is a group assessment. Use your join code to invite others to join the group.
-        A group must have <% if (groupConfig.maximum) { %>between <%= groupConfig.minimum %> and <%= groupConfig.maximum %><% } else { %>at least <%= groupConfig.minimum %><% } %> students.
+        A group must have <% if (groupConfig.minimum === groupConfig.maximum) { %><%= groupConfig.minimum %><% } else if (groupConfig.maximum) { %>between <%= groupConfig.minimum %> and <%= groupConfig.maximum %><% } else { %>at least <%= groupConfig.minimum %><% } %> students.
```

Comment:
Do we assert that `minimum` is present if `maximum` is? I'm just checking that these conditionals are sufficient.

Reply:
Line 10 has a check for a minimum larger than 1, so yes, minimum is always present in this line.

Reply:
Ah, great, thanks for pointing that out!